spec/services/capitals_creation_service_spec.rb

```diff
@@ -15,58 +21,104 @@
   end
 
   describe '.call' do
-    it 'creates bank accounts for this assessment' do
-      expect { subject }.to change { assessment.bank_accounts.count }.by(bank_accounts.count)
-      bank_accounts.each do |bank_account|
-        expect(assessment.bank_accounts.find_by!(name: bank_account[:name])[:lowest_balance].to_f).to eq(bank_account[:lowest_balance])
+    context 'with empty bank_accounts and non_liquid_capital' do
+      it 'returns an instance of CapitalCreationObject' do
+        expect(subject).to be_instance_of(described_class)
+      end
+
+      it 'does not create any capital item records' do
+        expect(assessment.capital_summary.capital_items).to be_empty
       end
     end
 
-    it 'creates non_liquid_assets for this assessment' do
-      expect { subject }.to change { assessment.non_liquid_assets.count }.by(non_liquid_assets.count)
+    context 'with liquid assets only' do
+      let(:bank_accounts) { liquid_assets_hash }
+
+      before { subject }
 
-      non_liquid_assets.each do |non_liquid_asset|
-        expect(assessment.non_liquid_assets.find_by!(description: non_liquid_asset[:description])[:value].to_f).to eq(non_liquid_asset[:value])
+      it 'creates liquid capital items' do
+        expect(capital_summary.liquid_capital_items.size).to eq 2
+        items = capital_summary.liquid_capital_items.order(:created_at)
+
+        expect(items.first.description).to eq bank_name_1
+        expect(items.first.value).to eq value_1
+        expect(items.last.description).to eq bank_name_2
+        expect(items.last.value).to eq value_2
       end
-    end
 
-    describe '#success?' do
-      it 'returns true' do
-        expect(subject.success?).to be true
+      it 'does not create non-liquid capital items' do
+        expect(capital_summary.non_liquid_capital_items).to be_empty
       end
     end
 
-    describe '#capital' do
-      it 'returns the created capital' do
-        expect(subject.capital[:bank_accounts].count).to eq(bank_accounts.count)
-        expect(subject.capital[:non_liquid_assets].count).to eq(non_liquid_assets.count)
-        expect(subject.capital[:bank_accounts].first.assessment.id).to eq(assessment.id)
-        expect(subject.capital[:non_liquid_assets].first.assessment.id).to eq(assessment.id)
+    context 'non_liquid_capital_items_only' do
+      let(:non_liquid_assets) { non_liquid_assets_hash }
+
+      before { subject }
+
+      it 'creates non liquid capital items' do
+        expect(capital_summary.non_liquid_capital_items.size).to eq 1
+        expect(capital_summary.non_liquid_capital_items.first.description).to eq item_1
+        expect(capital_summary.non_liquid_capital_items.first.value).to eq value_1
```

Comment:
do we also need a `it 'does not create liquid capital items'` test here, similar to the one above where we test that creating liquid capital items does not create non-liquid items?